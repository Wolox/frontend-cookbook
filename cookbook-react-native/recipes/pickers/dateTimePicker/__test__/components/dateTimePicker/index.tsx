import React from 'react';

import CustomDateTimePicker from '@pickersRecipes/dateTimePicker';

import renderer from 'react-test-renderer';

describe(' CustomDateTimePicker', () => {
  test('to match snapshot', () => {
    const component = renderer
      .create(<CustomDateTimePicker accessibilityLabel={''} is24hs={false} />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
