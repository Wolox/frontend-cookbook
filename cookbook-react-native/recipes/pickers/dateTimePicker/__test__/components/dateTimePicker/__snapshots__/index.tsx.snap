// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` CustomDateTimePicker to match snapshot 1`] = `
<View
  accessibilityLabel=""
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
      },
      undefined,
    ]
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignContent": "center",
        "alignItems": "center",
        "backgroundColor": "#FFF",
        "borderRadius": 20,
        "flexDirection": "row",
        "justifyContent": "center",
        "opacity": 1,
        "padding": 14,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#000",
            "fontSize": 16,
            "fontWeight": "bold",
          },
          undefined,
        ]
      }
    >
      CUSTOM_DATETIME_PICKER:OPEN
    </Text>
  </View>
  <Modal
    animationType="none"
    hardwareAccelerated={false}
    isVisible={false}
    onHide={[Function]}
    transparent={true}
    visible={false}
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "black",
          "bottom": 0,
          "height": 1334,
          "left": 0,
          "opacity": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "width": 750,
        }
      }
    />
  </Modal>
</View>
`;
